server.go

```diff
@@ -197,6 +198,43 @@ func HandleFunc(pattern string, handler func(ResponseWriter, *Msg)) {
 	DefaultServeMux.HandleFunc(pattern, handler)
 }
 
+// Writer writes DNS data frames; each call to Write should send an entire frame.
```

Comment:
I would just use DNS message as I use this term throughout the lib. Or maybe (raw) DNS message to distinguish them from the (parsed) DNS message.